dap_prinz_green_jobs/analysis/ojo_analysis/aggregate_by_soc.py

```diff
@@ -0,0 +1,82 @@
+from dap_prinz_green_jobs.analysis.ojo_analysis.process_ojo_green_measures import *
```

Comment:
ditto for all the functions imported this way 

```suggestion
import dap_prinz_green_jobs.analysis.ojo_analysis.process_ojo_green_measures as pg
```

Reply:
ditto adding a comment at the beginning to describe the script and how to run it

Reply:
good point!